x/gamm/pool-models/stableswap/amm_test.go

```diff
@@ -498,6 +498,48 @@ func TestCFMMInvariantMultiAssetsDirect(t *testing.T) {
 	}
 }
 
+func TestIterKCalculator(t *testing.T) {
```

Comment:
Unclear to me what this test is actually testing. What I'd test is the identity relation for `target_k` and `iter_k`

Reply:
I can do that if you are ok with it